### PR TITLE
Update dates for london event; add note to top of page w/ link to webinars

### PR DESCRIPTION
**Pull request by zchase**
This PR updates the dates for DevOpsCon London and adds a note to the top of the events page linking to our new webinar pages.

### CODE REVIEW

File: layouts/events/list.html [hunk 1]
```diff
@@ -13,7 +13,17 @@ <h1>Upcoming Events</h1>
...
     <div class="container mx-auto">
-        <div class="flex justify-between py-10 md:py-24 px-5 md:px-20">
+        <div class="w-10/12 mx-auto mt-10 note note-info" role="alert">
+            <p class="text-lg">
+                <i class="fas fa-info-circle pr-2"></i>
+                The demand for Pulumi content and education continues to grow and, even in
+                the face of restricted travel, we will ramp it up further. Although many events
+                worldwide we’d planned to sponsor or otherwise participate in have been canceled,
+                we have shifted to working with customers through digital channels wherever we can.
+                Checkout our <a href="{{ relref . "/webinars" }}"><b>upcoming webinars</b></a>.
```
Comment (cnunciato) [inline layouts/events/list.html:23]:
```suggestion
                Check out our <a href="{{ relref . "/webinars" }}"><b>upcoming webinars</b></a>.
```

File: layouts/events/list.html [hunk 2]
```diff
@@ -13,7 +13,17 @@ <h1>Upcoming Events</h1>
 
 {{ define "main" }}
     <div class="container mx-auto">
-        <div class="flex justify-between py-10 md:py-24 px-5 md:px-20">
+        <div class="w-10/12 mx-auto mt-10 note note-info" role="alert">
+            <p class="text-lg">
```
Comment (cnunciato) [inline layouts/events/list.html:17]:
```suggestion
            <p class="text-base">
```

File: layouts/events/list.html [hunk 3]
```diff
@@ -13,7 +13,17 @@ <h1>Upcoming Events</h1>
 
 {{ define "main" }}
     <div class="container mx-auto">
-        <div class="flex justify-between py-10 md:py-24 px-5 md:px-20">
+        <div class="w-10/12 mx-auto mt-10 note note-info" role="alert">
```
Comment (cnunciato) [inline layouts/events/list.html:16]: This doesn't quite seem to line up with the sections below it:

image

And needs padding to line up properly on mobile (we apply the same padding pretty much everywhere):

image

File: layouts/events/list.html (hunk 1 above)
Comment (cnunciato) [inline layouts/events/list.html:23]: I'd prefer without the bold, but if we want to make this stand out, we should use `<strong>` instead, 'cause semantic.

File: layouts/events/list.html (hunk 2 above)
Comment (zchase) [reply layouts/events/list.html:17]: 👍

File: layouts/events/list.html (hunk 1 above)
Comment (zchase) [reply layouts/events/list.html:23]: 👍

Comment (zchase) [reply layouts/events/list.html:23]: 👍

**Pull request merged**